src/esn.calendar.libs/app/services/fullcalendar/planning-render-event.service.spec.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+/* global sinon, chai: false */
+
+const expect = chai.expect;
+
+describe('The calFullCalendarPlanningRenderEventService service', function() {
+  var $q, element, session, calendar, event, tr, tds, a, title, fcTime, calUIAuthorizationService;
+
+  function Element() {
+    this.innerElements = {};
+    this.class = [];
+  }
+
+  Element.prototype.addClass = function(aClass) {
+    this.class.push(aClass);
+  };
+
+  Element.prototype.find = function(aClass) {
+    return this.innerElements[aClass];
+  };
+
+  Element.prototype.prepend = sinon.spy();
+
+  var userEmail = 'aAttendee@open-paas.org';
+
+  beforeEach(function() {
+    var emailMap = {};
+
+    emailMap[userEmail] = true;
+    session = {
+      user: {
+        emails: [userEmail]
+      }
+    };
+
+    calUIAuthorizationService = {
+      canModifyEvent: function() {
+        return true;
+      }
+    };
+
+    angular.mock.module('esn.calendar.libs');
+    angular.mock.module(function($provide) {
+      $provide.factory('session', function($q) {
+        session.ready = $q.when(session);
+
+        return session;
+      });
+      $provide.value('calUIAuthorizationService', calUIAuthorizationService);
+    });
+
+    event = {
+      attendees: []
+    };
+
+    element = new Element();
+    title = new Element();
+    fcTime = new Element();
+    a = new Element();
+    tr = new Element();
+    tds = new Element();
```

Comment:
What are these names stand for? Can we have explicit names?

Reply:
@tuanlc These represent the html tags, changing their name will break the test.  Is there any better way to make this clearer ?

Reply:
ok then